lib/ohai/plugins/openstack.rb

```diff
@@ -41,6 +42,27 @@ def openstack_dmi?
     end
   end
 
+  # if we're on windows, use wmi since it doesn't have dmi
+  def openstack_wmi?
+    if os == "windows"
+      require "wmi-lite"
```

Comment:
Note to self: poking around, I see I should be using `collect_os` here, and my require should be "wmi-lite/wmi"

Reply:
Or should it be a separate `collect_data(:windows)`?